test/functionalTest/cases/1027_get_all_entity_types/get_all_entity_types.test

```diff
@@ -38,6 +38,7 @@ brokerStart CB
 # 06. GET /v2/types and get a list of three types, each with one entity
 # 07. Create E4/T3/A4+A3
 # 08. GET /v2/types and get a list of three types, T3 now has two entities
+# 09. GET /v2/types wiht limit, getting 2 types but Fiware-Total-Count 3
```

Comment:
typo: wiht

I imagine this test is implemented already but was not in the 'TOC' ?